samples/custom-hostname-instance/createInstanceWithCustomHostname.js

```diff
@@ -0,0 +1,123 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Creates an instance with custom hostname.
+ *
+ * @param {string} projectId - ID or number of the project you want to use.
+ * @param {string} zone - Name of the zone you want to check, for example: us-west3-b
+ * @param {string} instanceName - Name of the new machine.
+ * @param {string} hostname - Custom hostname of the new machine.
+ *    Custom hostnames must conform to RFC 1035 requirements for valid hostnames.
+ * @param {string} machineType - Machine type you want to create in following format:
+ *    "zones/{zone}/machineTypes/{type_name}". For example:
+ *    "zones/europe-west3-c/machineTypes/f1-micro"
+ *    You can find the list of available machine types using:
+ *    https://cloud.google.com/sdk/gcloud/reference/compute/machine-types/list
+ * @param {string} sourceImage - Path the the disk image you want to use for your boot
```

Comment:
Typo: Path the the
I think you meant to say 'Path of the disk image you want to use...